### DISCUSSION TITLE
How to get version of data on Remote Configs from App

### DISCUSSION THREAD

**Discussion by waldobin** [invertase/react-native-firebase#7016] [Q&A] [answered]
Hi there, 
We use the remote config to fetch data from Firebase Cloud
`
import remoteConfig from '@react-native-firebase/remote-config';
.....
remoteConfig().getAll()
`
So are there any way to check the version of config from Admin Firebase dashboard into react-native-firebase logic app? Thank you

![ Admin Firebase dashboard](https://github.com/4astra/smart-pro-feb-2018/blob/master/Screenshot%202023-03-31%20at%2016.49.51.png "Admin Firebase dashboard")

**Comment by mikehardy** [comment, marked as answer]
Hi there! This information does not come through in any of the SDKs we reference (JS for API, android/ios for our implementations), it is simply not available.

The only way I could see to meet the use case would be to have an actual config value you call "configVersion" or similar, and increment it as a step in your project-specific process

**Comment by waldobin** [comment]
Hi Mike, thanks for getting back to me so quickly. I will update code as
suggestions. And hope your team will update this feature in the future.

**Reply by mikehardy** [reply]
@4astra please note, this is not in our control as the SDKs we wrap simply do not contain this information in any form. If this feature is of interest to you, you should go make a feature request following whatever process is outlined in the official firebase sites. If they implement it in their SDKs, we can wrap it and expose it here but not until then. Good luck with your project!
